mysite/blog/views.py

```diff
@@ -1,7 +1,7 @@
 from django.shortcuts import render, get_object_or_404
 from .models import Post
-from django.views.generic import ListView, FormView
-from .forms import EmailPostForm
+from django.views.generic import ListView ,FormView
+from .forms import EmailPostForm ,CommentForm
```

Comment:
your pre-commit black hook is not working. fix it

Reply:
[https://github.com/Karthikeyantestpress/Django_Blog_Application/pull/10/files#diff-ae5986e281ef46bd77930a4985d5c376ce41689b3f9b85dcae0e6b60c77eecc8:~:text=from%20.forms%20import%20EmailPostForm%2C%20CommentForm](url)

Reply:
fixed pre-commit issue